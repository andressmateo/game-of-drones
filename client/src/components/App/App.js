import React from 'react';
import { Switch, Route } from 'react-router-dom';
import styled from 'styled-components';

import { Register, Game, PlayersList } from '../../containers';

const Title = styled.div`
  text-align: center;
  width: 100%;
  height: 200px;
  padding-top: 100px;
  font-size: 70px;
`;

const Container = styled.div`
  display: flex;
  justify-content: center;
`;

const App = () => {
  return (
    <div>
      <Title>GAME OF DRONES</Title>
      <Container>
        <Switch>
          <Route exact path="/" component={Register} />
          <Route path="/game" component={Game} />
          <Route path="/players" component={PlayersList} />
        </Switch>
      </Container>
    </div>
  );
};

export default App;
