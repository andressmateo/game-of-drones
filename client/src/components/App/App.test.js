import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
import chai, { expect } from 'chai';
import { shallow } from 'enzyme';
import chaiEnzyme from 'chai-enzyme';
import chaiJestSnapshot from 'chai-jest-snapshot';

chai.use(chaiEnzyme());
chai.use(chaiJestSnapshot);

it('renders without crashing', () => {
  shallow(<App />);
});

it('renders correctly - snapshot', () => {
  const tree = renderer.create(<App />);
  expect(tree).to.matchSnapshot();
});
