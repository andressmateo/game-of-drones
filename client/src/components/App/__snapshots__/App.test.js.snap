// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly - snapshot 1`] = `
<div
  className="App"
>
  <div>
    <h1>
      No passwords :(
    </h1>
    <button
      className="more"
      onClick={[Function]}
    >
      Try Again?
    </button>
  </div>
</div>
`;
